src/functions-and-arrays.js

```diff
@@ -26,16 +48,69 @@ function sum() {}
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
-function averageNumbers() {}
+function averageNumbers(array) {
+  
+  if (array.length === 0) {
+    return null;
+  } else {
+    const average = sumNumbers(array) / array.length;
+  return average;
```

Comment:
indentation is important - try to match indentation - or even better, set up prettier and let it do the heavy lifting for you.